xwayland/xwm.c

```diff
@@ -238,13 +239,20 @@ static void xwm_set_net_client_list(struct wlr_xwm *xwm) {
 			XCB_ATOM_WINDOW, 32, mapped_surfaces, windows);
 }
 
+static void xsurface_set_net_wm_state(struct wlr_xwayland_surface *xsurface);
+
 static void xwm_send_focus_window(struct wlr_xwm *xwm,
```

Comment:
This function now not only sends focus events but also sets focus. Maybe rename to `xwm_set_focus_window`?